app/src/main/java/com/asfoundation/wallet/topup/TopUpLimitValues.kt

```diff
@@ -1,10 +1,16 @@
 package com.asfoundation.wallet.topup
 
 import com.asfoundation.wallet.ui.iab.FiatValue
+import com.asfoundation.wallet.util.Error
 import java.math.BigDecimal
 
 data class TopUpLimitValues(val minValue: FiatValue = INITIAL_LIMIT_VALUE,
-                            val maxValue: FiatValue = INITIAL_LIMIT_VALUE) {
+                            val maxValue: FiatValue = INITIAL_LIMIT_VALUE,
+                            val error: Error = Error()) {
+
+  constructor(isNoNetwork: Boolean) : this(INITIAL_LIMIT_VALUE, INITIAL_LIMIT_VALUE,
```

Comment:
`this(INITIAL_LIMIT_VALUE, INITIAL_LIMIT_VALUE,` is redundant since the primary constructor already have default values.
Please change it to `this(error = Error(true, isNoNetwork))`

Reply:
My intention was to remove the default values, but forgot